samples/azure-quantum/grover/Grover.ipynb

```diff
@@ -0,0 +1,710 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "# Simple Grover's Search with the Azure Quantum Service\n",
+    "\n",
+    "This notebook demonstrates how to use Q# and the Azure Quantum service together to search for data with Grover's algorithm, also known as amplitude amplification. By applying a sequence of reflections, this sample prepares a register of qubits in a state marked by a given quantum operation known as an oracle. The oracle used in this sample checks if its input matches a given integer, so that the computational basis state corresponding to that index is prepared with high probability."
```

Comment:
> also known as amplitude amplification

As a minor nit, Grover's algorithm is an example of amplitude amplification, but there are other examples as well. Definitely only a minor comment, though.

Reply:
Thank you, fixed!